cash/cash.c

```diff
@@ -0,0 +1,51 @@
+#include <cs50.h>
+#include <stdio.h>
+#include <math.h>
+int main(void)
+
+{
+    //type a non-nagetive value of cash owed, unit in $
+    float cash;
+    do
+    {
+      cash = get_float("Change owed: ");
+    }
+    while (cash < 0);
+
+    //print out the change owed
+//    printf("Change owed: %.2f\n", cash);
```

Comment:
Different levels of indentation throughout this file. Please make them consistent.